app/services/reinstate_conditions_met.rb

```diff
@@ -25,7 +25,7 @@ def save
         ActiveRecord::Base.transaction do
           ApplicationStateChange.new(application_choice).reinstate_conditions_met!
           application_choice.update(attrs)
-          UpdateOfferConditions.new(application_choice: application_choice).call
+          application_choice.offer.set_conditions_to_pending
```

Comment:
I think the pattern in Despo's PR is fine (and a bit simpler):
```
application_choice.offer.conditions.each(&:pending!)
```

Reply:
Ok i will amend